sympy/core/numbers.py

```diff
@@ -276,6 +276,8 @@ def _igcd2_python(a, b):
 
 
 # Use Lehmer's algorithm only for very large numbers.
+# The limit could be different on Python 2.7 and 3.x.
+# If so, then this could be defined in compatibility.py.
```

Comment:
I think this is a hangover from a merge. These comments were probably removed because Python 2 is no longer supported.